tests/core/test_lightning_module.py

```diff
@@ -379,3 +379,20 @@ def assert_device(device: torch.device) -> None:
     assert_device(torch.device("cpu"))
     trainer.predict(model, dataloaders=model.train_dataloader())
     assert_device(torch.device("cpu"))
+
+
+def test_on_before_backward(tmpdir):
```

Comment:
Update this test instead: `test_trainer_model_hook_system_fit`

Reply:
Done !